### PR TITLE
Fix CI run of SUSEConnect

### PR DESCRIPTION
**Pull request by digitaltom**
- fixed to build with clean buildroot
- fixed to replace files during package installation

See failures here: https://ci.suse.de/user/scc/my-views/view/scc-jobs/job/scc-Connect/

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/331682/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 20041951a17f7eca1a228bde0bb07e43dd66d325 into 559ee54849d0b20e2491f4249c2bea9d6ab4a3ca)

File: docker/integration.sh [hunk 1]
```diff
@@ -8,6 +8,6 @@ fi
 rake package:checkout &&
 rake package:build_gem &&
 rake package:generate_manpages &&
-cd package && osc build $PRODUCT x86_64 --no-verify --trust-all-projects && cd .. &&
+cd package && osc build $PRODUCT x86_64 --no-verify --trust-all-projects --clean && cd .. &&
```
Comment (felixsch) [inline docker/integration.sh:11]: Why is this clean needed? Maybe I missed the error in the CI logs.

Comment (digitaltom) [reply docker/integration.sh:11]: The error was not shown in the CI logs, because the process went interactive and was waiting for a user input.

Comment (digitaltom) [reply docker/integration.sh:11]: @felixsch like this abborted run: https://ci.suse.de/user/scc/my-views/view/scc-jobs/job/scc-Connect/1321/console 
it just appeared hanging for days.

Comment (felixsch) [reply docker/integration.sh:11]: Ah I see.

**Pull request merged**